docs/LwM2M_Client/Nordic/nRF9160DK.md

```diff
@@ -82,6 +82,10 @@ To connect the board:
 ## Configuring the Client
 
 0. With the board still connected to a serial port interface, open a serial communication program.
+
+    !!! note
```

Comment:
My idea is to move this part and connect with the next "note" part like:
`!!! note
        If your default credentials are different from the device credentials provided in {{ coiote_short_name }}, use the anjay stop command to stop LwM2M Client, and change them using the anjay config set <possible_option> <value> command.
        <br/>
        ![Anjay set configuration](images/anjay_config_set.PNG "Anjay set configuration"){:style="float: left;margin-right: 1177px;margin-top: 7px; margin-bottom: 17px;"}`